chart/k8gb/values.yaml

```diff
@@ -29,12 +29,16 @@ k8gb:
     level: info # log level (panic,fatal,error,warn,info,debug,trace)
   splitBrainCheck: false
   metricsAddress: "0.0.0.0:8080"
-  runAsUser: 1000
+  securityContext:
+    runAsNonRoot: true
+    readOnlyRootFilesystem: true
+    allowPrivilegeEscalation: false
+    runAsUser: 1000
 
 externaldns:
   image: k8s.gcr.io/external-dns/external-dns:v0.9.0
   interval: "20s"
-  securityContext:
+  securityContext: # For more options consult https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#podsecuritycontext-v1-core
```

Comment:
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#securitycontext-v1-core is actually more precise here

Reply:
good catch, the spec is almost the same :) also the link needs to be updated, because they will shutdown the old api docs eventually.. currently they host like 5 versions